README.md

```diff
@@ -66,6 +67,16 @@ target.
 
 ## Usage
 
+### Basic Playground
+
+You can see an example of Swish in action via the included `SwishExamples.playground`.
+
+To use that, clone this repository and run `carthage bootstrap`. When that
```

Comment:
Carthage used to build for all platforms by default — if this is still the case, could we add `--platform iOS` to match the platform the playground targets?